src/styles/progress-indicator.scss

```diff
@@ -0,0 +1,144 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-progress-indicator;
+$fd-progress-indicator-margin: 0.5rem 0 !default;
+$fd-progress-indicator-margin-mobile: 0.875rem 0 !default;
+$fd-progress-indicator-container-height: 1rem !default;
+
+$fd-progress-indicator-states: (
+  "informative": ("background": var(--fdProgress_Indicator_Background_Color_Informative), "icon": var(--fdProgress_Icon_Informative)),
+  "positive": ("background": var(--fdProgress_Indicator_Background_Color_Positive), "icon": var(--fdProgress_Icon_Positive)),
+  "negative": ("background": var(--fdProgress_Indicator_Background_Color_Negative), "icon": var(--fdProgress_Icon_Negative)),
+  "critical": ("background": var(--fdProgress_Indicator_Background_Color_Critical), "icon": var(--fdProgress_Icon_Critical))
+);
+
+.#{$block} {
+  width: 100%;
```

Comment:
`@include fd-reset()` please.

Reply:
It should not have a reset as progress indicator in all cases have truncation behaviour which is handled by a popover and in this case the markup is:
```
fd-progress-indicator fd-popover
```

If we bring a reset it will overwrite the rules coming from `fd-popover`.